Assets/Shopify/UIToolkit/Components/SelectionPicker.cs

```diff
@@ -0,0 +1,120 @@
+namespace Shopify.UIToolkit {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using UnityEngine.EventSystems;
+    using UnityEngine.Events;
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Interface for items that are used in the selection picker.
+    /// </summary>
+    public interface ISelectionPickerItem {
+        string Title { get; }
+    }
+
+    /// <summary>
+    /// A custom selection picker implementation.
+    /// </summary>
+    public class SelectionPicker : MonoBehaviour {
```

Comment:
Can this be accomplished by leveraging https://docs.unity3d.com/Manual/script-Dropdown.html?